app/views/providers/merits_task_lists/show.html.erb

```diff
@@ -1,22 +1,31 @@
 <%= page_template(
     page_title: t('.page_heading'),
-    back_link: { path: providers_legal_aid_application_capital_assessment_result_path(@legal_aid_application) }
+    back_link: { path: providers_legal_aid_application_capital_assessment_result_path(@legal_aid_application) },
+    show_errors_for: @merits_task_list
     ) do %>
 
-  <h2 class="govuk-heading-l"><%= t('.application_heading') %></h2>
-  <ol class="app-task-list">
-    <%= render 'task_list_section', name: t('.application_section_heading'), ccms_code: nil, tasks: @merits_tasks[:application], section_number: 1 %>
+  <div id='task_list' class="<%= 'govuk-form-group--error' if @merits_task_list&.errors&.any? && @merits_task_list.errors[:task_list].present? %>">
+    <h2 class="govuk-heading-l"><%= t('.application_heading') %></h2>
 
-    <h2 class="govuk-heading-l govuk-!-margin-top-4"><%= t('.proceeding_heading') %></h2>
-    <% @merits_tasks[:proceedings].each_with_index do |(_, proceeding), index| %>
-      <%= render 'task_list_section', name: proceeding[:name], tasks: proceeding[:tasks], ccms_code: @merits_tasks[:proceedings].keys[index], section_number: index + 2 %>
+    <% if @merits_task_list&.errors&.any? && @merits_task_list.errors[:task_list].present? %>
+      <span class="govuk-error-message" id="task-list-error">
+        <span class="govuk-visually-hidden">Error: </span>
+        <%= @merits_task_list&.errors[:task_list]&.first %>
```

Comment:
are these safe operators required here? I dont think code wont be executed if @merits_task_list.errors[:task_list].first

Reply:
I think I added them for a reason, but that may have changed later. I'll try removing it and see what happens! 

I don't want this blocking stuff much longer 👍

Reply:
In that case I dont feel strongly enough about them that they have to be removed

Reply:
Double checked and you were right.  I think the model, originally, couldn't handle errors so it was a separate error model for a while until I refactored it back into the form. :+1: